indexer/editable_map_object.cpp

```diff
@@ -9,6 +10,51 @@
 #include "std/cctype.hpp"
 #include "std/cmath.hpp"
 
+
+namespace
+{
+bool PushLanguage(StringUtf8Multilang const & names, int8_t const index, vector<osm::LocalizedName> & result)
+{
+  if (StringUtf8Multilang::kUnsupportedLanguageCode == index ||
+      StringUtf8Multilang::kDefaultCode == index)
+    return false;
```

Comment:
Please, add curly braces around ```return false```, as your if statement doesn't fit to a single line.